src/example_generated.rs

```diff
@@ -0,0 +1,330 @@
+//! This module contains an example of code generated by the macro. **IT IS NOT MEANT
+//! TO BE USED OUTSIDE THIS CRATE**.
+//!
+//! If you would like to see what the actual code looks like, click the `src` button
+//! located top-right. If you are not interested in the code and simply want to
+//! browse the functionality exposed to you, click on the `ExampleFlags` struct and
+//! read on.
+
+use __core::cmp::Ordering;
+use __core::hash::{Hash, Hasher};
```

Comment:
Use statements on line 9 and 10 here are not used. Marking so that I remember to remove them.